packages/spectrum/test/util.tsx

```diff
@@ -34,20 +34,39 @@ import {
 } from '@jsonforms/core';
 import { JsonForms } from '@jsonforms/react';
 import { spectrumRenderers } from '../src/index';
+import {
+  defaultTheme,
+  Provider as SpectrumThemeProvider,
+} from '@adobe/react-spectrum';
+import { act } from 'react-dom/test-utils';
 
-export const mountForm = (
+type OnChangeType<T> = (change: { errors: any; data: T }) => void;
+
+// tslint:disable-next-line:only-arrow-functions
+export function mountForm<T extends object>(
   uischema: UISchemaElement,
   schema: JsonSchema = {},
-  data: any = {}
-): ReactWrapper => {
+  data?: T,
+  cells: any = [],
```

Comment:
Kleines Detail, das könntest du etwas spezifischer als `JsonFormsCellRendererRegistryEntry[]` typen (wie in `JsonForms.cells`)